src/examples/java/software/amazon/nio/spi/examples/WalkFromRoot.java

```diff
@@ -29,13 +38,13 @@ public static void main(String[] args) throws IOException {
 
 
         var root = Paths.get(URI.create(bucketName));
-        System.err.println("root.getClass() = " + root.getClass());
+        logger.info("root.getClass() = {}", root.getClass());
 
         var s3 = root.getFileSystem();
 
         for (var rootDir : s3.getRootDirectories()) {
             try (var pathStream = Files.walk(rootDir)) {
-                pathStream.forEach(System.out::println);
+                pathStream.forEach((p) -> logger.info(p.toString()));
```

Comment:
can probably be simplified to `forEach(logger::info)`

Reply:
No 😞 `p` is a `Path`, not a `String`. And `logger.info` takes a `String`